test/e2e/gke_node_pools.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
-var _ = framework.KubeDescribe("GKE node pools [Feature:GKENodePool]", func() {
+var _ = framework.SIGDescribe("node", "GKE node pools [Feature:GKENodePool]", func() {
```

Comment:
I would say it's more cluster-lifecycle thing

Reply:
the current agreement is that GKE test specifics fall under SIG Cloud Provider (`/area gcp`)
SIG CL tries to be product agnostic.

Reply:
Shall we set this to cloud-provider then?

Reply:
yeah - ok - cloud-provider is better then

Reply:
raised separate PR per sig tag 
 - sig-cloud-provider https://github.com/kubernetes/kubernetes/pull/99086